src/components/Greeting.js

```diff
@@ -0,0 +1,15 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchMessage } from '../redux/features/greetingSlice';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const greeting = useSelector((state) => state.greeting.text);
+  useEffect(() => {
+    // console.log('useEffect is running');
+    dispatch(fetchMessage());
+  }, [dispatch]);
+  return <h1>{greeting}</h1>;
+};
```

Comment:
## [OPTIONAL]

- [ ] You're doing it well! I've noticed that the isLoading state is managed in Redux, but it seems the loading indicator isn't displayed in the component. Considering user experience, adding the loading indicator in the component itself could enhance visual feedback.

Just a thought for optimizing the user interface! Keep up the good work, and feel free to discuss if you have any considerations around this.